src/components/Form/CrossPodWriteForm.jsx

```diff
@@ -97,51 +107,106 @@ const CrossPodWriteForm = () => {
       statusType="Upload status"
       defaultMessage="To be uploaded..."
     >
-      <form onSubmit={handleCrossPodUpload} autoComplete="off">
-        <div style={formRowStyle}>
-          <label htmlFor="cross-upload-doc">Upload document to username: </label>
-          <br />
-          <br />
-          <input
-            id="cross-upload-doc"
-            size="25"
-            name="crossPodUpload"
-            value={username}
-            onChange={(e) => setUsername(e.target.value)}
-            placeholder={selectedUser.username}
-          />
-        </div>
-        <div style={formRowStyle}>
+      <Box display="flex" justifyContent="center">
+        <form onSubmit={handleCrossPodUpload} autoComplete="off">
+          <FormControl fullWidth>
+            {/* TODO: Determine whether Typography below is necessary or redundant */}
+            {/* Either way, also determine whether to update all other forms to that */}
+            <Typography htmlFor="cross-upload-doc">Upload document to username:</Typography>
+            <TextField
+              id="cross-upload-doc"
+              name="crossPodUpload"
+              label="Search Username"
+              required
+              value={username}
+              onChange={(e) => setUsername(e.target.value)}
+              placeholder={selectedUser.username}
+            />
+          </FormControl>
           <DocumentSelection
             htmlForAndIdProp="upload-doc"
             handleDocType={handleDocType}
             docType={docType}
           />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="upload-doc-expiration">Expiration date (if applicable): </label>
-          <input id="upload-doc-expiration" name="date" type="date" />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="upload-doc-desc">Enter description: </label>
           <br />
+          <FormControl>
+            {/* TODO: Determine whether InputLabel below is necessary or redundant */}
+            <InputLabel htmlFor="upload-doc-expiration" />
+            <LocalizationProvider dateAdapter={AdapterDayjs}>
+              <DatePicker
+                id="upload-doc-expiration"
+                name="date"
+                format="MM/DD/YYYY"
+                label="Expiration Date"
+                value={expireDate}
+                onChange={(newExpireDate) => setExpireDate(newExpireDate)}
+                type="date"
+                slotProps={{
+                  textField: {
+                    helperText: 'MM/DD/YYYY'
+                  }
+                }}
```

Comment:
Think you could replace Lines 144 and 148 with just `<FormHelperText>MM/DD/YYYY</FormHelperText>` after  `<DatePicker>` like what we've done with for `FormControl` for the "Choose file" button , so something along the lines of:

```
<FormControl>
  <LocalizationProvider>
    <DatePicker />
    <FormHelperText>MM/DD/YYYY</FormHelperText>
  </LocalizationProvider>
</FormControl>
```